compiler/src/iree/compiler/Modules/IO/Parameters/Transforms/test/import_parameters.mlir

```diff
@@ -0,0 +1,36 @@
+// RUN: iree-opt --pass-pipeline="builtin.module(iree-io-export-parameters{path="opt=%t.irpa" minimum-size=0},iree-io-import-parameters{paths="opt=%t.irpa"})" %s | FileCheck %s
```

Comment:
Checking my understanding: this exports _then_ imports, round tripping?

I suppose we could also test with `iree-create-parameters`, if we didn't want to rely on the sequencing there. Feels sneaky to me to have the files needed by the compiler not all already available on disk when the command begins.

Reply:
round-tripping is best - it's more sneaky to have runtime deps in compiler lit tests :)

Reply:
Ah, good point. SGTM